x-pack/plugins/cloud_security_posture/public/pages/rules/index.tsx

```diff
@@ -86,49 +36,37 @@ export const Rules = ({ match: { params } }: RouteComponentProps<PageUrlParams>)
                 <EuiButtonEmpty iconType="arrowLeft" contentProps={{ style: { padding: 0 } }}>
                   <FormattedMessage
                     id="xpack.csp.rules.rulesPageHeader.benchmarkRulesButtonLabel"
-                    defaultMessage="Benchmark Rules"
+                    defaultMessage="Benchmarks"
                   />
                 </EuiButtonEmpty>
               </Link>
             </EuiFlexItem>
             <EuiFlexItem>
-              <CloudPosturePageTitle
-                title={i18n.translate('xpack.csp.rules.rulePageHeader.pageHeaderTitle', {
-                  defaultMessage: 'Rules - {integrationName}',
-                  values: {
-                    integrationName: packageInfo?.name,
-                  },
-                })}
-              />
+              <EuiFlexGroup gutterSize="s">
+                <EuiFlexItem grow={false} style={{ marginBottom: `6px` }}>
```

Comment:
really minor but in JSS you dont have to use pixels as they are the default value if you just pass numbers.

meaning this will work the same:
```suggestion
                <EuiFlexItem grow={false} style={{ marginBottom: 6 }}>
```